crates/ra_syntax/src/ast/traits.rs

```diff
@@ -105,6 +106,12 @@ pub trait AttrsOwner: AstNode {
     }
 }
 
+pub trait AsyncOwner: AstNode {
```

Comment:
this could just be a normal method on `FnDef` (in `extensions.rs`)?

Reply:
I just think It will be useful for async block syntax for future. But I could just add it in FnDef if you prefer.

Reply:
I'm not convinced it's worth the abstraction, but I'm fine with either way :slightly_smiling_face:

Reply:
Then lets me remove it 😄